packages/gatsby-cli/src/create-cli.js

```diff
@@ -182,6 +182,16 @@ function buildLocalCommands(cli, isLocalSite) {
           default: false,
           describe: `Build site without uglifying JS bundles (for debugging).`,
         })
+        .option(`log-pages`, {
+          type: `boolean`,
+          default: false,
+          describe: `Log the pages that changes since last build (for optimization).`,
+        })
+        .option(`write-to-file`, {
+          type: `boolean`,
+          default: false,
+          describe: `Save the log of changed pages for future comparison (for optimization).`,
+        })
```

Comment:
Those switches should be available only when `GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES` is used or description should mention that. Those don't do anything in default builds

Reply:
Let's just use `hidden: true` on them for hot fix - this should ensure same behaviour as before they broke with `yargs` upgrade. We can later discuss if this is or isn't best idea